lib/raft/tests/messages_comparator.hpp

```diff
@@ -0,0 +1,16 @@
+#include <CppUTest/TestHarness.h>
+#include <CppUTestExt/MockSupport.h>
+#include "../raft.hpp"
+#include "test_state_machine.hpp"
+
+class RaftMessageComparator : public MockNamedValueComparator
+{
+public:
+    virtual bool isEqual(const void* object1, const void* object2);
+    virtual SimpleString valueToString(const void* object);
```

Comment:
Consider implementing `operator==` and `operator<<` for `Message` if that makes sense